CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ option(ENABLE_FORTRAN77 "Build the Fortran 77 bindings" OFF)
 option(ENABLE_JAVA      "Build Java bindings" OFF)
 option(ENABLE_CXX       "Build C++ bindings" OFF)
 option(ENABLE_APPS      "Build utility applications" OFF)
+option(ENABLE_INSTALL_NXTRANSLATE_LIBS "Install libraries for nxtranslate" OFF)
```

Comment:
I don't understand the point of this option! you need it on if you turn ENABLE_APPS on !